src/main/kotlin/de/bund/digitalservice/useid/config/SecurityConfig.kt

```diff
@@ -25,7 +26,7 @@ class SecurityConfig(
     @Bean
     fun securityFilterChain(http: HttpSecurity): SecurityFilterChain {
         return http.authorizeHttpRequests()
-            .requestMatchers("$IDENTIFICATION_SESSIONS_BASE_PATH/*/tc-token").permitAll()
+            .requestMatchers("$TCTOKEN_BASE_PATH/*").permitAll()
```

Comment:
Can be removed (hopefully, please validate).

Reply:
That's right. 
This made me thinking if we shouldn't require authentication by default and just explicitly set `permitAll` on public endpoints?

Reply:
But out of scope for this PR probably.

Reply:
Good point, I like that from a security perspective. Since we only have very few endpoints with authentication it might be less code for a list like this. But we should take a look at this - after migrating all the endpoints and not in the scope of this PR, I'd say.